SECURITY.md

```diff
@@ -0,0 +1,67 @@
+# Security
+
+## Reporting a Vulnerability
+
+If you think you have found a potential security vulnerability in `checkov`,
+please email info@bridgecrew.io directly. Do not file a public issue. If
```

Comment:
can you please set the address to security@bridgecrew.io

Reply:
Updated